example/plopfile.js

```diff
@@ -153,29 +162,32 @@ module.exports = function (plop) {
 				name: 'fileName',
 				message: 'Pick a file name:',
 				validate: function (value) {
-					if ((/.+/).test(value)) { return true; }
+					if (/.+/.test(value)) {
+						return true;
+					}
 					return 'file name is required';
 				}
-			}, {
+			},
+			{
 				type: 'directory',
 				name: 'path',
 				message: 'where would you like to put this component?',
 				basePath: plop.getPlopfilePath()
 			}
 		],
 		actions: [
-			function(data) {
+			function (data) {
```

Comment:
function is not a method so should have a psace

Reply:
Again: Let's let prettier do what prettier wants. Comments like these otherwise are not very helpful as they're entirely preferential.

Reply:
Again: it was done by prettier.